src/convert.ts

```diff
@@ -1042,3 +1038,26 @@ const convertSpellHigherLevels = (ddbSpell: DdbSpell): AlchemySpellAtHigherLevel
   }
 }
 */
+
+const convertTrackers = (ddbCharacter: DdbCharacter): AlchemyTracker[] => {
+    return [
+        {
+            name: 'XP',
+            category: 'experience',
+            color: 'Yellow',
+            max: 355000,
```

Comment:
doesn't necessarily need to hold up this PR, but I imagine the most common way people use the XP tracker is to track XP until their next level (rather than theoretical max XP at lv20, or something). seems like all we'd need to do to support that is to have a static map between levels and XP counts, and then use the character's level to put the correct total here.

Reply:
Good point! And yeah, we actually do this for characters created in Alchemy, too. I could yoink that function from our code.

Reply:
Alright, I went head and pulled in those functions from the Alchemy 5e implementation and updated the way the XP tracker is created so that it tracks XP until next level now.